spec/views/form/page_view_spec.rb

```diff
@@ -22,8 +22,6 @@ def assign_attributes(object, attrs)
       Singleton.__init__(FormHandler)
       example.run
     end
-    Timecop.return
-    Singleton.__init__(FormHandler)
```

Comment:
@bibblobcode similarly here Jack, you may not be aware that a while ago @kosiakkatrina put a line in the test config so that the FormHandler singleton is reinitialised after every single test, so anywhere you see this you can also try removing it as it should be redundant